library/pkparse.c

```diff
@@ -545,7 +546,18 @@ static int pk_get_rsapubkey( unsigned char **p,
 
     if( ( ret = mbedtls_rsa_import_raw( rsa, *p, len, NULL, 0, NULL, 0,
                                         NULL, 0, NULL, 0 ) ) != 0 )
-        return( MBEDTLS_ERR_PK_INVALID_PUBKEY );
+    {
+        if( ret == MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED )
```

Comment:
I'm a bit lost. When would a function from the `rsa` module return `MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED` vs `MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED | MBEDTLS_ERR_RSA_xxx`?

Reply:
Why would there be two low level errors? I don't see a place where`MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED | MBEDTLS_ERR_RSA_xxx` is returned.
The previous behavior was returning ` MBEDTLS_ERR_PK_INVALID_PUBKEY` for *any* failure, now, in case of `MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED` returned by the rsa module, we still return `MBEDTLS_ERR_PK_INVALID_PUBKEY` but we also add the low level `MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED`